tests/common/fixtures/advanced_reboot.py

```diff
@@ -408,8 +408,8 @@ def __runPtfRunner(self, rebootOper=None):
         '''
         logger.info("Running PTF runner on PTF host: {0}".format(self.ptfhost))
 
-        prebootOper = rebootOper if rebootOper is not None and 'routing' in rebootOper else None
-        inbootOper = rebootOper if rebootOper is not None and 'routing' not in rebootOper else None
+        prebootOper = rebootOper if rebootOper is not None and 'routing' not in rebootOper else None
+        inbootOper = rebootOper if rebootOper is not None and 'routing' in rebootOper else None
```

Comment:
Please add a comments here to explain the calculation of preboot and inboot.

Reply:
Done

Reply:
OK